girder/constants.py

```diff
@@ -29,6 +29,11 @@
 STATIC_PREFIX = os.path.join(sys.prefix, 'share', 'girder')
 STATIC_ROOT_DIR = os.path.join(STATIC_PREFIX, 'static')
 
+# Server modes
+PRODUCTION_MODE = 'production'
```

Comment:
I know this seems minor, but could we scope these under a class like `ServerMode`, then have `ServerMode.PRODUCTION`, etc.?

I'd like to avoid further spamming this namespace with more flat values. However, if you disagree, feel free to push back on this suggestion.

Reply:
I agree, I'll make those changes